src/style.css

```diff
@@ -0,0 +1,96 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&display=swap');
+
+body {
+  background-color: #f6f6f6;
+  font-family: 'Open Sans', sans-serif;
+}
+
+#list-container {
+  width: 80%;
+  max-width: 600px;
+  background-color: #f6f6f6;
+  box-shadow: 0 2px 8px rgb(0 0 0 / 15%);
+}
+
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  background-color: white;
+}
+
+li {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
```

Comment:
### OPTIONAL
- [ ] It would be better if we implement the CSS DRY principle to avoid the repetition of the  properties that makes our codebase hard to maintain